test/logr.test.js

```diff
@@ -404,6 +402,10 @@ describe('logr', () => {
       expect(lastMessage).to.equal('[tag1] message1');
     });
   });
+
+  describe('multiple reporters', () => {
```

Comment:
History.  I reorganized the plugin unit tests to all be in one group but left the old one there.